deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,572 @@
oZiGbjRDG06Xsbv+5WwJAAAAADi2aeh6K6zndHnxbXTv2QIAh/X18x9ek5Ly8dMXI7yPzyWAr6tA/nWIr6MAAO4jLEzAhbUUEwAAAJXeVQRc2nF2vwQAAAAArvwhujqcH7xnCwB4TQqsfw85mwF8XQVSigkAANxHWNajCQAAAAAAAAAAAAAAAKhcmAB2p5gAAACAvRBwAQAAAAAAAAAAWlFMAAAAcFi+KzY41wAAAFAtARcAAAAAAAAAAAAAAABaECYA5xkAAABqJOACAAAAAAAAAAC0IkwAAADgXgg4zwAAAFAbARcAAAAAAAAAAKAJ09CNVgAAADi03gTgPAMAAECNBFwAAAAAAAAAAAAAAABoRTEBNC9MAAAAwN4IuAAAAAAAAAAAAC0oJgAAAOBB+AGcYwAAAKiQgAsAAAAAAAAAAAAAAACtGE0AzetNAAAAwN4IuAAAAAAAAAAAAC0IEwAAAPCvYgJoVpgAAACAPRJwAQAAAAAAAAAAqjcNne+wDgAAwH96E0CzwgQAAADskYALAAAAAAAAAAAAAAAArQkTQJMEmAAAANglARcAAAAAAAAAAKB2xQQAAAD8IkwAzi0AAADUQsAFAAAAAAAAAAAAAACA1vQmgOYUEwAAALBXAi4AAAAAAAAAAEDtwgQAAAA8o5gAmjKaAAAAgL0ScAEAAAAAAAAAAKo2DZ0P9wAAAPCcMAE0o5gAAACAPRNwAQAAAAAAAAAAahYmAAAA4AWCn9COMAEAAAB7JuACAAAAAAAAAADULEwAAADAK4oJoAmCSwAAAOyagAsAAAAAAAAAAAAAAACtChNA9YoJAAAA2DsBFwAAAAAAAAAAoFrT0PVWAAAA4BXjg4gL1M4ZBQAAYPcEXAAAAAAAAAAAAAAAAGhZmACqNpoAAACAvRNwAQAAAAAAAAAAahUmAAAA4Aa9CaBaxQQAAAAcgYALAAAAAAAAAABQqzABAAAA7pDgbAIAAEDtBFwAAAAAAAAAAIBahQkAAAC4UTEBVGk0AQAAAEcg4AIAAAAAAAAAAFRpGjof8AEAAOBW7pBQn2ICAAAAjkLABQAAAAAAAAAAqFGYAAAAgDsVE0BVehMAAABwFAIuAAAAAAAAAABAjcIEAAAAuEuC8wgAAAAtEHABAAAAAAAAAAAAAABgD8YH0QiohbMIAADAoQi4AAAAAAAAAAAA1ZmGrrcCAAAACWECqIL3dgAAADgUARcAAAAAAAAAAAAAAAD2QjQCthcmAAAA4GgEXAAAAAAAAAAAgNoUEwAAAPAOYQJwBgEAAGBNAi4AAAAAAAAAAAAAAADsSZgANtWbAAAAgKMRcAEAAAAAAAAAAKoyDZ0P+QAAAPAe7pWwnTABAAAARyTgAgAAAAAAAAAAAAAAwN4UE8AmwgQAAAAckYALAAAAAAAAAABQk2ICAAAAZhAmgE30JgAAAOCIBFwAAAAAAAAAAAAAAADYm9EEsLowAQAAAEcl4AIAAAAAAAAAAFRjGjrfpRkAAIC5FBOAMwcAAABrEHABAAAAAAAAAAAAAABgj0RCYV2jCQAAADgqARcAAAAAAAAAAKAWxQQAAADMLEwAqygmAAAA4MgEXAAAAAAAAAAAAAAAANirMAE4awAAALC0JxMAAAAAAAAAAAA1mIaut0J1vybP/v3TZTQOAADQiu93zWIGWJw3CwAAADi0RxMAAAAAAAAAAAAVCBMAAACwkGICcMYAAABgSQIuAAAAAAAAAABADcIEAAAAuHOCMwYAAAAtEnABAAAAAAAAAABqECYAAABgIaMJwBkDAACAJQm4AAAAAAAAAAAAm5uGzgd9AAAAWFIxAThbAAAAsBQBFwAAAAAAAAAAYGthAgAAANw9wdkCAACAVgm4AAAAAAAAAAAAWwsTAAAAsLDR/RMWO1sAAABweAIuAAAAAAAAAADA1sIEAAAAuH9Cc4oJAAAA4B8CLgAAAAAAAAAAwKamofOdmgEAAFhDbwKYVZgAAAAA/iHgAgAAAAAAAAAAbKmYAAAAgBWFCWC2syTKCwAAAP8ScAEAAAAAAAAAAAAAAOAoiglgFmECAAAA+D8BFwAAAAAAAAAAYDPT0PVWAAAAYEWjCWAW3tMBAACAHwi4AAAAAAAAAAAAWwkTAAAAsIFiAniXMAEAAAD8TMAFAAAAAAAAAADYSpgAAAAA91FwhgAAAKB1TyYAAAAAAAAAAAA20p8u4yL/4mnorAsAAMBLvl9G4/o4mwJSehMAAADAzx5NAAAAAAAAAAAAAAAAwMGECcDZAQAAgLkIuAAAAAAAAAAAAFsoJgAAAGBDvQkgJUwAAAAAvxNwAQAAAAAAAAAAthAmAAAAwN0UmiN+BAAAAM8QcAEAAAAAAAAAALYwmgAAAICNhQnAmQEAAIA5CLgAAAAAAAAAAABrKyYAAACgAr0J4C5hAgAAAHiegAsAAAAAAAAAALC2MAEAAACVKCaAm4keAQAAwAsEXAAAAAAAAAAAgLWNJgAAAKASYQK4STEBAAAAvEzABQAAAAAAAAAAWFOYAAAAgIqIjMJtwgQAAADwMgEXAAAAAAAAAABgTWECAAAAKlNMAG8SOwIAAIBXCLgAAAAAAAAAAABr6k0AAACAuyo0pZgAAAAAXifgAgAAAAAAAAAArCVMAAAAgDsrOB8AAACwNwIuAAAAAAAAAADAWsIEAAAAuLNCc0YTAAAAwOsEXAAAAAAAAAAAgLX0JgAAAMCdFZpSTAAAAABvE3ABAAAAAAAAAADWEGv9RNPQWRsAAICMYgL4TZgAAAAA3ibgAgAAAAAAAAAArCFMAAAAgLsrNGc0AQAAALxNwAUAAAAAAAAAAFhDbwIAAAAqJ1QBPysmAAAAgNsIuAAAAAAAAAAAAEsrJgAAAMAdFpoTJgAAAIDbCLgAAAAAAAAAAABLCxMAAADgDgvNnYXRDAAAAHAbARcAAAAAAAAAAGBpPuwDAABAS3fYMAM4BwAAAHAPARcAAAAAAAAAAGBJxQQA
```

Comment:
I didn't know we will need to duplicate all of this.
Its size is 68KB~ (4464×1944), pretty big for an icon.
Could you do one of the following:
- lower its dimensions (keeping the ratio)
- find a similar, smaller image
so it would be around 200×100?

Reply:
please open ticket for me, I'll consider your  suggestions there.

Reply:
Done - CSI-1204.